docs/docs-ref-conceptual/requirement-sets/excel-api-requirement-sets.md

```diff
@@ -30,9 +31,283 @@ For more information about versions, build numbers, and Office Online Server, se
+|[style](/javascript/api/excel/excel.style)|_Property_ > textOrientation|The text orientation for the style.|1.8|
+|[subtotals](/javascript/api/excel/excel.subtotals)|_Property_ > automatic|If Automatic is set to true, then all other values will be ignored when setting the Subtotals.|1.8|
+|[subtotals](/javascript/api/excel/excel.subtotals)|_Property_ > average|If Automatic is set to true, then all other values will be ignored when setting the Subtotals.|1.8|
+|[subtotals](/javascript/api/excel/excel.subtotals)|_Property_ > count|If Automatic is set to true, then all other values will be ignored when setting the Subtotals.|1.8|
+|[subtotals](/javascript/api/excel/excel.subtotals)|_Property_ > countNumbers|If Automatic is set to true, then all other values will be ignored when setting the Subtotals.|1.8|
+|[subtotals](/javascript/api/excel/excel.subtotals)|_Property_ > max|If Automatic is set to true, then all other values will be ignored when setting the Subtotals.|1.8|
+|[subtotals](/javascript/api/excel/excel.subtotals)|_Property_ > min|If Automatic is set to true, then all other values will be ignored when setting the Subtotals.|1.8|
+|[subtotals](/javascript/api/excel/excel.subtotals)|_Property_ > product|If Automatic is set to true, then all other values will be ignored when setting the Subtotals.|1.8|
+|[subtotals](/javascript/api/excel/excel.subtotals)|_Property_ > standardDeviation|If Automatic is set to true, then all other values will be ignored when setting the Subtotals.|1.8|
+|[subtotals](/javascript/api/excel/excel.subtotals)|_Property_ > standardDeviationP|If Automatic is set to true, then all other values will be ignored when setting the Subtotals.|1.8|
+|[subtotals](/javascript/api/excel/excel.subtotals)|_Property_ > sum|If Automatic is set to true, then all other values will be ignored when setting the Subtotals.|1.8|
+|[subtotals](/javascript/api/excel/excel.subtotals)|_Property_ > variance|If Automatic is set to true, then all other values will be ignored when setting the Subtotals.|1.8|
+|[subtotals](/javascript/api/excel/excel.subtotals)|_Property_ > varianceP|If Automatic is set to true, then all other values will be ignored when setting the Subtotals.|1.8|
+|[table](/javascript/api/excel/excel.table)|_Property_ > legacyId|Returns a numeric id. Read-only.|1.8|
+|[workbook](/javascript/api/excel/excel.workbook)|_Property_ > readOnly|True if the workbook is open in Read-only mode. Read-only.|1.8|
+|[workbookCreated](/javascript/api/excel/excel.workbookcreated)|_Property_ > id|Returns a value that uniquely identifies the WorkbookCreated object. Read-only.|1.8|
+|[workbookCreated](/javascript/api/excel/excel.workbookcreated)|_Method_ > [open()](/javascript/api/excel/excel.workbookcreated)|Open the workbook.|1.8|
+|[worksheet](/javascript/api/excel/excel.worksheet)|_Property_ > gridlines|Gets or sets the worksheet's gridlines flag.|1.8|
```

Comment:
gridlines and showGridlines have the same description - are they the same? ditto for headings and showHeadings.

Reply:
Those don't exist either.

Reply:
@AlexJerabek @ElizabethSamuel-MSFT gridlines was renamed showGridlines and headings was renamed showHeadings. We have a snippet that demos Worksheet.showGridlines